services/brig/test/unit/Test/Brig/Calling.hs

```diff
@@ -323,7 +323,7 @@ testSFTStaticV2NoStaticUrl = do
       . ignoreLogs
       . interpretSFTInMemory mempty
       . throwErrorInIO @_ @NoTurnServers
-      $ newConfig env (Discovered turnUri) Nothing (Just sftEnv) (Just . unsafeRange $ 2) ListAllSFTServers (CallsConfigV2 Nothing)
+      $ newConfig env (Discovered turnUri) Nothing (Just sftEnv) (Just . unsafeRange $ 2) ListAllSFTServers (CallsConfigV2 Nothing) True
```

Comment:
Perhaps add a test with `False` here and test that `s` is changed accordingly. I was admittedly sloppy when I adjusted these tests last time. 😞